Tiltfile

```diff
@@ -18,7 +18,7 @@ if advanced_go_dev_mode:
 
     local_resource(
         'gitops-server',
-        'GOOS=linux GOARCH=amd64 make gitops-server',
+        'GOOS=linux make gitops-server',
```

Comment:
Why was it even there (GOARCH)? :O that would mean amd64 was used on newer mac laptops too (M1 chip)